packages/interface/src/components/interface-skeleton/style.scss

```diff
@@ -120,6 +120,23 @@ html.interface-interface-skeleton__html-container {
 	}
 }
 
+.is-distraction-free {
+
+	.interface-interface-skeleton__secondary-sidebar,
+	&.is-sidebar-opened .interface-interface-skeleton__sidebar {
+		border: $border-width solid $gray-200;
+		margin: $grid-unit-15;
+		border-radius: 2px;
+	}
+
+	.interface-interface-skeleton__body {
+		margin-top: -60px;
+
+		.interface-interface-skeleton__content {
+			padding-top: 60px;
```

Comment:
It would be nice to do something the awkward gap this creates when the document background isn't white:

https://user-images.githubusercontent.com/846565/161947831-d8ebd30b-4d0d-431f-9e15-330ce99ddd81.mp4

Reply:
I assumed this wasn't going to work as easily :D Maybe we can copy the bg color, or extend the actual content, at least in the post-editor.

Reply:
Yeah applying the background to `.edit-post-layout` or something could work for now. 

It's probably important to keep the gap, to account for selecting the very first block when there's no internal padding on the document.

Reply:
I wonder If the toolbar should reposition below the block instead of needing this gap

Reply:
A similar issue exists with the sidebars as well :D 

<img width="591" alt="Screenshot 2022-04-06 at 11 47 34" src="https://user-images.githubusercontent.com/846565/161958475-96870b95-f951-4a98-8aba-1cb96faaeb16.png">

Reply:
It'd be nice to try and make them float like popovers in this mode

Reply:
This gap thing is easily solved by positioning the header as absolute and giving it 100vw width. See https://github.com/WordPress/gutenberg/pull/38928/commits/b9ecd0504ad136ff4f14e1255fad1ca0ad133ed7 (also cc @shaunandrews as this commit shows persistent sidebars as well).

Reply:
https://user-images.githubusercontent.com/107534/166667685-5fd3cf41-04b9-4d2a-a880-37428d93c581.mp4

@shaunandrews here is the compromise: a somewhat ugly margin on floating chrome. Otherwise there is a z-index war between the top chrome and the side chrome which wither makes the sidebars impossible to close or the top bar's buttons inaccessible. 

Hmm there would be an option to fiddle with `z-index` on hover/focus?